scripts/verify-install.sh

```diff
@@ -122,20 +137,20 @@ for signature in $SIGNATURES; do
   echo ""
 
   # Then make sure that the hash of the installed binaries can be found in the
-  # signed list of hashes.
-  if ! grep -q "$LND_SUM" "$signature"; then
-    echo "ERROR: Hash $LND_SUM for lnd not found in $signature: "
+  # manifest.
+  if ! grep -q "$LND_SUM" "$MANIFEST"; then
+    echo "ERROR: Hash $LND_SUM for lnd not found in $MANIFEST: "
```

Comment:
cat of $signature below, should be manifest. Here and other places

Reply:
Fixed.